src/features/user/Profile/components/MyTrees/MyTrees.tsx

```diff
@@ -186,7 +187,7 @@ export default function MyTrees({ profile, authenticatedType, token }: Props) {
             </div>
             <div className={styles.myTreesContainer}>
               <div className={styles.treesList}>
-                {contributions.map((contribution, index) => {
+                {contributions.map((contribution: any, index: any) => {
```

Comment:
While using array methods, typescript infers the type pretty well if it is provided at the top level.
Since array `contributions` have already been given a type, you don't need to define type for parameters in map.
So could you remove `any` for both `contribution` and `index`.